### PR TITLE
Add the connector common keys - EI-618

### PR DESCRIPTION
**Pull request by mihai-dinculescu**

### CODE REVIEW

File: src/properties.rs [hunk 1]
```diff
@@ -21,6 +21,8 @@ pub mod common_keys {
         pub const HOST_ALLOW_LIST_PROPERTY: &str = "http://data.iotics.com/public#hostAllowList";
 
         // portal
+        pub const SPACE_NAME_PROPERTY: &str = "https://data.iotics.com/app#spaceName";
+        pub const COLOR_PROPERTY: &str = "https://data.iotics.com/app#color";
```
Comment (Elleppi) [inline src/properties.rs:25]: Not an error, but in British English it's COLOUR

**Review by Elleppi — APPROVED** [review]
LGTM

File: src/properties.rs (hunk 1 above)
Comment (mihai-dinculescu) [reply src/properties.rs:25]: Good point. But the value also contains color and we'd have to change the portal in lockstep. I'd rather do that separately.

Comment (Elleppi) [reply src/properties.rs:25]: It's not a problem to me

Comment (mihai-dinculescu) [reply src/properties.rs:25]: https://ioticlabs.atlassian.net/browse/EI-670

**Pull request merged**